src/components/ReloadPrompt.tsx

```
@@ -0,0 +1,59 @@
+          ) : (
+            <span>
+              New content available, click on reload button to update.
+            </span>
+          )}
+        </div>
+        <div className='flex gap-4'>
+          {needRefresh && (
+            <button
+              className='ReloadPrompt-toast-button border border-toxic-green px-3 py-1 hover:bg-toxic-green hover:text-void-black transition-colors focus-visible:ring-toxic-green focus-visible:ring-offset-void-black'
+              onClick={() => updateServiceWorker(true)}
+            >
```

Biome (2.4.13)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)